unity-renderer/Assets/Scripts/MainScripts/DCL/DataStore/DataStore_HUDs.cs

```diff
@@ -34,7 +34,8 @@ public class DataStore_HUDs
         public readonly BaseVariable<bool> shouldShowNotificationPanel = new BaseVariable<bool>(true);
         public readonly BaseVariable<Transform> notificationPanelTransform = new BaseVariable<Transform>(null);
         public readonly BaseVariable<Transform> topNotificationPanelTransform = new BaseVariable<Transform>(null);
-        public readonly BaseVariable<bool> isSceneUIEnabled = new BaseVariable<bool>(true);
+        public readonly BaseVariable<bool> isCurrentSceneUiEnabled = new BaseVariable<bool>(true);
+        public readonly BaseDictionary<int, bool> isSceneUiEnabled = new ();
```

Comment:
quick questions, sorry, I'm having troubles to understand it.
why are 2 variables required to handle scene's ui visibility?
where or under what conditions is `BaseDictionary<int, bool> isSceneUiEnabled` being modified? I couldn't find it

Reply:
At first, i tried setting `isCurrentSceneUiEnabled` but only works for the current scene. It doesn't work for portable experiences, it seems they don't become the current scene. In order to disable/enable the UI for the PX specifically, i need to find the scene's root element in the hierarchy.
The variable is being set in `ExperiencesViewerComponentController` whenever the user clicks the hide/show portable experience UI button.

Reply:
I see. Thanks.
I would suggest using just one variable to toggle ui visibility, `isCurrentSceneUiEnabled` could iterate loaded scenes and set the visibility value for non-global scenes and also set them when scenes a new scene is loaded.
But I don't feel that should be a blocker, so I'll leave the suggestion at your discretion if you see it fit and if you got the time

Reply:
I did analyze this solution and i think that the current one is better. Its harder to get a consistent state of the visibility of each scene by managing all of them in `isSceneUiEnabled`. Now i can override the visibility state of a specific scene. If by any means `isCurrentSceneUiEnabled` changes, its more complicated to keep the visibility state of a PX.